Libraries/LibWeb/MediaCapabilitiesAPI/MediaCapabilities.h

```diff
@@ -1,15 +1,107 @@
 /*
  * Copyright (c) 2024, Jamie Mansfield <jmansfield@cadixdev.org>
+ * Copyright (c) 2025, Psychpsyo <psychpsyo@gmail.com>
  *
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
 #pragma once
 
 #include <LibWeb/Bindings/PlatformObject.h>
+#include <LibWeb/EncryptedMediaExtensions/EncryptedMediaExtensions.h>
 
 namespace Web::MediaCapabilitiesAPI {
 
+// https://w3c.github.io/media-capabilities/#dictdef-videoconfiguration
+struct VideoConfiguration {
+    String content_type;
+    unsigned long width;
+    unsigned long height;
```

Comment:
should these two (and the double below) be WebIDL:: prefixed types instead of raw C types? I'm not sure how consistent we've been on this with new IDLs tbh.

Reply:
Probably?
I've changed them over now cause that just seems like the correct thing to do.